src/glibc.zig

```diff
@@ -681,6 +685,26 @@ pub const BuiltSharedObjects = struct {
 
 const all_map_basename = "all.map";
 
+fn wordDirective(target: std.Target) []const u8 {
+    // Based on its description in the GNU `as` manual, you might assume that `.word` is sized
+    // according to the target word size. But no; that would just make too much sense.
+    return switch (target.cpu.arch) {
```

Comment:
```suggestion
    return switch (target.ptrBitWidth()) {
```

?

Reply:
I think I left it like this because I was unsure about whether ILP32 ABIs should use `.word`. I guess it doesn't really matter either way since I'll have to audit this when I start working on support for those.